src/libais/ais.h

```diff
@@ -1566,9 +1566,49 @@ class Ais8_200_10 : public Ais8 {
   Ais8_200_10(const char *nmea_payload, const size_t pad);
 };
 
-// 21 and 22 do not exist
+// vessel_traking_and_tracing_standard_ed1-2_ccnr_23-apr_2013_en.pdf
+// ETA at lock/bridge/terminal
+// Todo : complete this class
```

Comment:
Use Google TODO style:   "Todo :" -> `TODO(#issuenum)` or `TODO(pyrog):`

And say what is missing from the class rather than just that it is incomplete.

Reply:
Ooops, I think I added all the fields to the class, so I must remove the TODO.
Thanks for explanations on Google TODO style 😄